modules/nuxeo-apidoc-webengine/src/main/resources/skin/views/distribution/stats.ftl

```diff
@@ -0,0 +1,143 @@
+            <thead>
+              <tr>
+                <th class="select">
+                  <input type="checkbox" id="selectAll">
+                </th>
+                <th class="header">
+                  Distribution Name
+                </th>
+                <th class="header">
+                  Version
+                </th>
+              </tr>
+            </thead>
+            <tbody>
+              <#list Root.listPersistedDistributions() as distrib>
+              <tr>
+                <td class="select">
+                  <input type="checkbox" name="distrib" id="${distrib.key}" value="${distrib.key}">
+                </td>
+                <td>
+                  <label for="${distrib.key}">${distrib.name}</label>
+                </td>
+                <td>
+                  <label for="${distrib.key}">${distrib.version}</label>
+                </td>
+              </tr>
+              </#list>
+            </tbody>
+          </table>
+          <input type="button" class="submitSelection" value="Submit" />
+        </form>
+      </div>
+      <div class="stats statcharts">
+        <div id="errors" class="chartError"></div>
+        <div id="feedback" class="chartHeader"></div>
+        <div class="charts">
+          <div class="chart main" id='codeTypeStats'></div>
+          <div class="chart details" id='codeTypeStatsDetails'></div>
+          <div style="clear:both;"></div>
+        </div>
+        <div class="charts">
+          <div class="chart main" id='xpStats'></div>
+          <div class="chart details" id='xpStatsDetails'></div>
+          <div style="clear:both;"></div>
+        </div>
+        <div class="charts">
+          <div class="chart main" id='studioStats'></div>
+          <div class="chart details" id='studioStatsDetails'></div>
+          <div style="clear:both;"></div>
+        </div>
+      </div>
+      <div style="clear:both;"></div>
+    </div>
+  </div>
+
+</@block>
+
+<@block name="footer_scripts">
+  <script>
+    const rootURL = '${Root.path}';
+    const exporterName = 'jsonContributionStats';
+    const loadingDataText = "Loading data";
+    const noDataText = "No data";
+
+    const render = () => {
+      $("#feedback").html(loadingDataText);
+      $("#errors").html('');
+
+      var checkboxes = document.querySelectorAll('input[name="distrib"]:checked');
+      var distribKeys = [...checkboxes].map(c => c.value);
+      var total = distribKeys.length;
+      if (total == 0) {
+        $("#feedback").html(noDataText);
+        $("#codeTypeStats").html('');
+        $("#xpStats").html('');
+        $("#studioStats").html('');
+        $("#codeTypeStatsDetails").html('');
+        $("#xpStatsDetails").html('');
+        $("#studioStatsDetails").html('');
+      } else {
+        var allStats = [];
+        distribKeys.map(d => Plotly.d3.json(rootURL + '/' + d + '/export/' + exporterName, (err, data) => renderDistrib(err, data, d, total, allStats)));
```

Comment:
nit: I'd use template strings `${rootURL}/${d}....`

Reply:
Strangely that did not work ok (replacements were always empty) so I gave up trying to understand why